common/utilities/string/wrap-text.h

```diff
@@ -0,0 +1,117 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include <string>
+#include <vector>
+#include <sstream>
+
+#include "../third-party/imgui/imgui.h"
+
+namespace utilities {
+namespace string {
+
+
+// Split input test to lines
+//     - Input: std:string text
+//     - Output: a vector of (std::string) lines
+// Example:
+//     Input:
+//          This is the first line\nThis is the second line\nThis is the last line
+//     Output:
+//          [0] this is the first line
+//          [1] this is the second line
+//          [2] this is the last line
+inline std::vector< std::string > split_string_by_newline( const std::string & str )
```

Comment:
Maybe just call it `split`? And pass the delimiter in?

Reply:
Then you can `split` it into another .h :)

Reply:
👍